xcconfigs/Shopist.xcconfig

```diff
@@ -0,0 +1,4 @@
+#include "Datadog.xcconfig"
+
+SHOPIST_BASE_URL=some.url
+#include? "Shopist.local.xcconfig" // overrides SHOPIST_BASE_URL
```

Comment:
Can we 🙏 include this file in project navigator, so it's easily accessible? It will be red when not created, which is also good for understanding that it's missing. As we do in SDK project:

<img width="221" alt="Screenshot 2020-08-03 at 11 31 15" src="https://user-images.githubusercontent.com/2358722/89168232-d8e88700-d57c-11ea-86e4-02fef507d901.png">